blueprints/modular.yaml

```diff
@@ -0,0 +1,31 @@
+title: Modular
+'@extends': default
+
+form:
+  fields:
+    tabs:
+      fields:
+        buttons:
+          type: tab
+          title: Modular
+          fields:
+            header.modules_in_menu:
+              type: toggle
+              label: Submenu for modules
+              highlight: 1
+              description: If this is activated, each child module will be a navigable submenu item under the parent page in the navbar.
+              options:
+                1: Yes
+                0: No
+              validate:
+                type: bool

```

Comment:
Let's put new lines between the fields just to make this easier to read.

Reply:
Agreed. I havent really paid attention to the formatting so far. I will go over all the docs and do a pass of formatting the files.